logstash-input-beats.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "input" }
 
   # Gem dependencies
-  s.add_runtime_dependency "logstash-core", ">= 2.0.0", "< 3.0.0"
+  s.add_runtime_dependency "logstash-core", ">= 2.0.0", "< 6.0.0"
```

Comment:
we need to ensure alpha1 here. We don't know if 6.0 will break backwards compatibility. Keep it as is means that a logstash beta2 deployment of 6.0.0 can still install these plugins.